docker/hq-compose.yml

```diff
@@ -35,7 +35,8 @@ postgres:
     - ${VOLUME_PREFIX}postgresql:/var/lib/postgresql/data
 
 couch:
-  image: klaemo/couchdb:1.6.1
+  image: klaemo/couchdb:2.0-dev
```

Comment:
doesn't look like the official 2.0 image is published yet: https://hub.docker.com/r/library/couchdb/